kumascript/macros/ObsoleteGeneric.ejs

```diff
@@ -24,9 +25,9 @@ if ($1) {
         n = string.trim(string.substr(t, 2));
         ver = "JavaScript ";
         if (string.length(n) > 0) {
-            if(wiki.pageExists('/' + lang + '/docs/JavaScript/New_in_JavaScript/' + n)) {
+            if(true) {
```

Comment:
ditto

Reply:
Ya, seems I missed a few. Will update it